### ISSUE TITLE
Rebinding `M-p` makes it impossible to navigate the history

### ISSUE THREAD

**Issue by egarrulo** [gempesaw/ido-vertical-mode.el#30]
`ido-completing-map` binds `M-p` to `previous-history-element`.  IDO Vertical Mode rebinds this key, thus making it impossible to navigate the history.

**Comment by gempesaw** [comment]
```elisp
(setq ido-vertical-define-keys 'C-n-C-p-up-down-left-right)
```

that'll let you use <kbd>left</kbd> and <kbd>right</kbd> to navigate the history. 

I don't use the history nav much, but I can definitely see the motivation for wanting to customize that if you're interested in filing a PR.

**Comment by gempesaw** [comment]
closed by #31
